integrationExamples/gpt/userId_example.html

```diff
@@ -160,7 +160,7 @@
                         },
                         storage: {
                             type: "cookie",
-                            name: "_parrable_eid", // create a cookie with this name
+                            name: "_parrable_id", // create a cookie with this name
```

Comment:
given that this is user-configurable, how should we be handling this upgrade @justinegreene ?
should a new prebid.js build that expects the composite cookie do a storage.name check before attempting to parse its contents? my thought is that if a client upgrades prebid but does not configure the new storage.name then we'll be trying to read `_parrable_eid` as the composite cookie.